### DISCUSSION TITLE
Install server in Onprem env.

### DISCUSSION THREAD

**Discussion by lyonelf** [amidaware/tacticalrmm#1500] [General]
**Server Info (please complete the following information):**
 - OS: Debian 11
 - Browser: 
 - RMM Version (as shown in top left of web UI): last 

**Installation Method:**
  - [X ] Standard
  - [ ] Docker

**Agent Info (please complete the following information):**
- Agent version (as shown in the 'Summary' tab of the agent from web UI):
- Agent OS: [e.g. Win 10 v2004, Server 2012 R2]

**Describe the bug**
When i launch the script ./install.sh  i have an error ( cf picture)

I'd like to install TRMM locally, no need to have the server on the web expose

**Comment by dinger1986** [comment]
hello, what the error?

**Comment by lyonelf** [comment]
Oups sorry, the upload has failed

Hereis :

![2023-05-04_12h27_51](https://user-images.githubusercontent.com/34097487/236198921-a4494b89-0f43-46b1-8ea6-92b67d3bb0ab.png)


But after lunch it's clear.... i don't have sudo on my server ...

**Reply by dinger1986** [reply]
do you have sudo installed? Are you running as root?

You can install using `apt install sudo`

**Comment by lyonelf** [comment]
Yes i installed it and it's ok now.

Next step ;)

Thanks

**Comment by lyonelf** [comment]
2 others options missing
![2023-05-04_14h31_13](https://user-images.githubusercontent.com/34097487/236204721-1dd5c549-111f-4e29-849f-a38087f48b48.png)

![2023-05-04_14h31_03](https://user-images.githubusercontent.com/34097487/236204738-80edfaf6-8198-4ac3-b601-6deb38c07fda.png)

**Reply by dinger1986** [reply]
Seems like your Debian 11 install is missing a lot of basics like cp, grep sudo etc

**Comment by silversword411** [comment]
Agreed with dinger...re-read the install docs, new VM and follow all the steps exactly

**Comment by lyonelf** [comment]
I restarted from scratch and now it's Ok, i've the " installation complete!"

And the 2FA prompt after login console

**Reply by dinger1986** [reply]
check here https://docs.tacticalrmm.com/troubleshooting/#cant-login-on-server-after-first-setup

Check your DNS (you have used a real domain?)

**Comment by lyonelf** [comment]
I've modifed the post, it's now ok !

Thanks for fast answers
